lib/pbench/cli/server/pbench_create_user.py

```diff
@@ -32,18 +39,21 @@ def execute(self):
             click.echo(f"User {self.context.username} registered")
             return 0
 
+        click.echo(response.json()["message"])
+
         if self.context.exists_ok and response.status_code == 403:
-            click.echo(f"User {self.context.username} already registered")
             return 0
 
-        click.echo(
-            payload["message"] if "message" in payload else response.reason, err=True
-        )
         return 1
 
 
 @click.command()
-@common_options
+@click.option(
+    "--config-file",
+    required=False,
+    default=os.environ.get("_PBENCH_SERVER_CONFIG"),
+    help="Specify the config file",
+)
```

Comment:
Hmm; I like having the `@common_options` here. One option is to simply clone and edit `lib/cli/agent/options.py` to `lib/cli/server` so that we can use it on the server side as well. A better but messier option would be to refactor this to lift it outside of the `agent`/`server` subtree and share it for both (though having the separate config classes makes this a bit tricker we could raise most of the code with reference to subclass helpers supplied by either side.)

Reply:
I like the idea have having a uniform user interface across all of our tools, to the extent that we can achieve it.  The best way to accomplish that is by using common code.

However, for the `-C`/`--config-file` switch, I don't think the interface is actually common:  for agent-side tools, `-C` should specify an _agent_ config file (and, if the switch is missing, it should consult the `_PBENCH_AGENT_CONFIG` environment variable for the value), whereas for server-side tools, `-C` should specify a _server_ config file (and, if the switch is missing, it should probably consult some other environment variable).  (And, if it's a "client-side" tool, it should probably have a third behavior variant, which, possibly, devolves to one or both of the previous....)

So, I guess we just go with the clone approach.  :-p

Reply:
Fixed it, by cloning the options.py file from agent implementation and making the appropriate edits